expr/ztests/type-map.yaml

```diff
@@ -5,7 +5,7 @@ zed: |
     "conn": conn,
     "dns": dns
   }|
-  split (
+  * | split (
```

Comment:
🤮

Reply:
This problem is already there... the bug fix uncovered it.
:-)